build.gradle

```
@@ -18,6 +18,7 @@ repositories {
 }
 
 dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
     implementation 'org.springframework.boot:spring-boot-starter-jdbc'
```

```suggestion
    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
```
spring-boot-starter-jdbc 는 제거하시면 좋겠습니다~
사용하지 않은 패키지가 쌓이면 나중엔 사용여부를 분간하기 어려워집니다ㅎㅎ